common/samd21g18a_firmware_partition.ld

```diff
@@ -49,7 +49,7 @@ SEARCH_DIR(.)
 /* Memory Spaces Definitions */
 MEMORY
 {
-  rom    (rx)  : ORIGIN = 0x00001000, LENGTH = 0x00008000
+  rom    (rx)  : ORIGIN = 0x00001000, LENGTH = 0x0003efff
```

Comment:
SAMD2118a has more memory (256kb compared to 64kb) so we have to increase the length of the rom section (the first 0x1000 bytes are reserved for the bootloader).